PENDING_RELEASE_NOTES.md

```diff
@@ -3,7 +3,12 @@
 _Date TBD_
 
 ### New Features
+* Added ROLE_ONC user role. ROLE_ONC replaces the existing ROLE_ADMIN and all relevant accounts were converted to the new role.
```

Comment:
Weren't these first few new lines already deployed to PROD?

Reply:
Update notes